lib/src/benchmark.dart

```diff
@@ -57,7 +57,7 @@ class BenchmarkResult {
     if (units != null) {
       output
         ..labeled('units', units)
-        ..labeled('units/second', runsPerSecond)
+        ..labeled('units/second', runsPerSecond / units)
```

Comment:
thanks for the PR but this change isn't right: would result in the following:
```
          total runs:   16 095   
          total time:   2.0000  s
         average run:      124 μs
         runs/second:  8 064.5   
               units:   10 000   
        units/second:   0.8065   
       time per unit:   0.0124 μs
```

while it's supposed to be
```
HashMap[k]
          total runs:   15 737   
          total time:   2.0000  s
         average run:      127 μs
         runs/second:  7 874.0   
               units:   10 000   
        units/second: 78 740 157   
       time per unit:   0.0127 μs
```

I've pushed the fixed version

Reply:
Oops. Fixed